src/Components/Sidebar/Sidebar.js

```diff
@@ -1,13 +1,42 @@
 import React from 'react';
 import './Sidebar.scss';
 import SidebarProject from '../SidebarProject/SidebarProject';
+import { retrieveProjects } from '../../util/apiCalls';
+// import { useSelector, useDispatch } from 'react-redux';
+// import { useEffect } from 'react';
+import { connect } from 'react-redux';
+import { addProjects } from '../../actions';
 
-export const Sidebar = () => {
-  return (
-    <> 
-      <SidebarProject />
-    </>
-  )
+
+export class Sidebar extends React.Component {
+  componentDidMount() {
+    retrieveProjects()
+      .then(data => {
+        this.props.addProjects(data)
+      })
+  }
+
+  render(){
+    return (
+      <>
+        {this.props.projects.map(project => {
+        return (
+          <SidebarProject
+            title={project.title}
+          />
+        )
```

Comment:
Can you change this part to match the comp?  So, Projects will be at the top of the Sidebar, with an underline (you'll need to make a little div for this, then just show the border on one side), then each title will be rendered after that.  The way that you have it now, I think each title will have Projects: [title name].